src/web_connection_base.cpp

```diff
@@ -113,7 +113,11 @@ namespace libtorrent
 
 	void web_connection_base::start()
 	{
-		set_upload_only(true);
```

Comment:
actually, I don't see where check_invariant() is called from when calling set_seed(), in RC_1_1